<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APIStatusUsage</class>
 <widget class="QWidget" name="APIStatusUsage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_status_good_icon">
       <property name="text">
        <string>&lt;okay_icon&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status_good">
       <property name="text">
        <string>Okay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status_bad_icon">
       <property name="text">
        <string>&lt;error_icon&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status_bad">
       <property name="text">
        <string>Error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status_offline_icon">
       <property name="text">
        <string>&lt;offline_icon&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status_offline">
       <property name="text">
        <string>Not Connected</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_refresh">
       <property name="toolTip">
        <string>Refresh</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_usage_label">
     <property name="text">
      <string>Usage:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_usage_field" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_usage_error_icon">
        <property name="text">
         <string>&lt;error_icon&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_usage_warn_icon">
        <property name="text">
         <string>&lt;warn_icon&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_usage">
        <property name="text">
         <string>&lt;usage_placeholder&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_usage">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
